### DISCUSSION TITLE
Create Multiple Dashboards for Partners

### DISCUSSION THREAD

**Discussion by fxmb** [plausible/analytics#2060] [Anything Else]
Is there a feature planned to create multiple dashboards for different partners who want to track the traffic they contribute etc?

=> We would need a possibility to preset certain filters, e.g. `utm_source` and let the partner see all the traffic coming from their source?

**Comment by metmarkosaric** [comment]
thanks for the feedback @fxmb! would be nice to have some [saved filters](https://github.com/plausible/analytics/discussions/1495) and then you could link to a specific saved filter. at the moment, you could try and explore our [stats API](https://plausible.io/docs/stats-api) as it allows you to build a dashboard from scratch

**Reply by fxmb** [reply]
Thanks for the fast reply. 

Yes, preset filters would be nice, it would be important that people opening the link cannot unset the filters to see more data, e.g. in our use case just the partner can see their specific data but no one elses.
